extensions/scala-web/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "scala",
+  "displayName": "Scala Syntax (official)",
+  "description": "Official Scala Syntax",
+  "version": "0.5.0",
+  "publisher": "scala-lang",
+  "license": "SEE LICENSE IN LICENSE.md",
+  "engines": {
+    "vscode": "^1.5.0"
+  },
+  "homepage": "https://github.com/scala/vscode-scala-syntax/blob/master/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scala/vscode-scala-syntax.git"
+  },
+  "icon": "images/smooth-spiral.png",
+  "categories": [
+    "Programming Languages"
+  ],
+  "contributes": {
+    "languages": [
+      {
+        "id": "scala",
+        "aliases": [
+          "Scala",
+          "scala"
+        ],
+        "extensions": [
+          ".scala",
+          ".sbt",
+          ".sc"
+        ],
+        "configuration": "./language-configuration.json"
+      }
+    ],
+    "grammars": [
+      {
+        "language": "scala",
+        "scopeName": "source.scala",
+        "path": "./syntaxes/Scala.tmLanguage.json"
+      }
+    ]
+  },
+  "devDependencies": {
+    "@types/node": "^14.6.4",
+    "ajv": "^6.12.4",
+    "github-changes": "^1.1.1",
+    "npm-run-all": "^4.1.5",
+    "rimraf": "^3.0.2",
+    "ts-node": "^9.0.0",
+    "typescript": "^4.0.2",
+    "vsce": "^1.79.5",
+    "vscode-tmgrammar-test": "0.0.10"
+  },
+  "scripts": {
+    "clean": "rimraf scala-*.vsix",
```

Comment:
You must add a `compile` command. Otherwise, it would fail the build https://github.com/conwnet/github1s/blob/master/scripts/build/build-github1s-extensions.sh#L13

Reply:
I just have to add?:

```
"scripts":
{
    "compile": "echo done",
    "watch": "echo done"
}
```

Reply:
fixed this with #83